pkg/schedule/checker/affinity_checker.go

```diff
@@ -34,21 +38,27 @@ import (
 	"github.com/tikv/pd/pkg/utils/logutil"
 )
 
+const recentMergeTTL = time.Minute
+
 // AffinityChecker groups regions with affinity labels together by affinity group.
 // It ensures regions adhere to affinity group constraints by creating operators.
 type AffinityChecker struct {
 	PauseController
-	cluster         sche.CheckerCluster
-	affinityManager *affinity.Manager
-	conf            config.CheckerConfigProvider
+	cluster          sche.CheckerCluster
+	affinityManager  *affinity.Manager
+	conf             config.CheckerConfigProvider
+	recentMergeCache *cache.TTLUint64
+	startTime        time.Time
```

Comment:
Does it only set this value in the construct?

Reply:
Yes, consider pd transfer leader, so we also pause schedule 1 min